flask/main.py

```diff
@@ -178,18 +178,37 @@ def connect():
 @app.route('/product/0/info', methods=['GET'])
 def product_info():
     time.sleep(.55)
+    return "/product/0/info"
```

Comment:
will fix this error that we are seeing a bunch which messes with our crash-free session rate
https://testorg-az.sentry.io/issues/3966708105/?project=5808655&referrer=release-issue-stream
<img width="1334" alt="image" src="https://user-images.githubusercontent.com/314061/228994976-d585f7dc-2e71-45ff-80c1-e54014e2ce49.png">

<img width="1161" alt="image" src="https://user-images.githubusercontent.com/314061/228994909-7045d13a-b14b-4490-9e60-72533a024cda.png">